File: App/app/src/main/java/com/team1/covid19info/model/AdviseQuestion.java
package com.team1.covid19info.model;

public class AdviseQuestion {
    private int linkYesAnswer;
    private int linkNoAnswer;
    private String questionText;
    private String adviceText;

    public AdviseQuestion( String body, String advice) {
        this.questionText = body;
        this.adviceText = advice;
    }

    public AdviseQuestion( int linkYesAnswer, int linkNoAnswer, String body) {
        this.linkYesAnswer = linkYesAnswer;
        this.linkNoAnswer = linkNoAnswer;
        this.questionText = body;
    }


    public int getLinkYesAnswer() {
        return linkYesAnswer;
    }

    public int getLinkNoAnswer() {
        return linkNoAnswer;
    }

    public String getQuestionText() {
        return questionText;
    }

    public String getAdviceText() {
        return adviceText;
    }
}

File: App/app/src/main/java/com/team1/covid19info/model/AdviseQuestion.kt
package com.team1.covid19info.model

class AdviseQuestion {
    var linkYesAnswer: Int = 0
    var linkNoAnswer: Int = 0
    var questionText: String? = null
    var adviceText: String? = null
    var infoButton: Boolean = false

    constructor(body: String, advice: String, info: Boolean) {
        this.questionText = body
        this.adviceText = advice
        this.infoButton = info
    }

    constructor(linkYesAnswer: Int, linkNoAnswer: Int, body: String) {
        this.linkYesAnswer = linkYesAnswer
        this.linkNoAnswer = linkNoAnswer
        this.questionText = body
    }
}
